CHANGELOG.asciidoc

```diff
@@ -3,6 +3,210 @@
 :issue: https://github.com/elastic/beats/issues/
 :pull: https://github.com/elastic/beats/pull/
 
+[[release-notes-7.13.0]]
+=== Beats version 7.13.0
+https://github.com/elastic/beats/compare/v7.12.1...v7.13.0[View commits]
+
+==== Breaking changes
+
+*Affecting all Beats*
+
+- Use alias to report container image in k8s metadata. {pull}24380[24380]
```

Comment:
@ChrsMark is this a breaking change?

Reply:
Yes it is for the reasons described at https://github.com/elastic/beats/pull/24380#issuecomment-794098920.